.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,17 +6,11 @@
 
 
 
-# FE Screenshots (If Applicable)
-
-
-
 <details>
     <summary><h4>Author Checklist</h4></summary>
 
-- [ ]  Can a reviewer skim this PR / Is this a non code change?
+- [ ]  Non code change
 - [ ]  Followed DRY (Don't repeat yourself)
-- [ ]  Migrations Required/Applied?
```

Comment:
I do tick this one every now and then :)

Reply:
I've seen you use it! I'm of two minds about keeping it though, since it's a fundamental of Django, so it's useful to remind the author of a PR to ensure they've not missed anything obvious, but I expect that if the author had missed migrations, then it'd be one of the the first things caught in the review. Thinking about it while I write this comment, I'm inclined to agree that it should be there, if for nothing else but the philosophy that each engineer is responsible for the quality of the work they submit for PR, in the long run it'll save time.